keps/sig-release/1731-publishing-packages/kep.yaml

```diff
@@ -2,6 +2,7 @@ title: Publishing kubernetes packages
 kep-number: 1731
 authors:
   - "@hoegaarden"
+  - "@RobertKielty"
```

Comment:
I think this may go into `editor`. Please add all reviewers to the list below as well.